sql/src/main/java/io/crate/analyze/SetStatementAnalyzer.java

```diff
@@ -40,23 +40,23 @@
     private SetStatementAnalyzer() {}
 
     public static SetAnalyzedStatement analyze(SetStatement node, ParameterContext parameterContext) {
-        if (SetStatement.Scope.SESSION.equals(node.scope())) {
-            logger.debug("SET SESSION STATEMENT: {}", node.toString());
+        if (!SetStatement.Scope.GLOBAL.equals(node.scope())) {
+            logger.warn("SET {} STATEMENT: {}", node.scope().toString(), node.toString());
```

Comment:
The node.toString() already prints out the scope, so printing it again is not necessary